### PR TITLE
Refactored composer.json

### PR DESCRIPTION
**Pull request by svycka**
now uses PSR-4 and added test and check commands

### CODE REVIEW

File: src/SlmLocale/Module.php [hunk 1]
```diff
@@ -44,27 +44,14 @@
...
 use Zend\ModuleManager\Feature;
 use Zend\Mvc\MvcEvent;
 use Zend\Stdlib\ResponseInterface;
 
 class Module implements
-    Feature\AutoloaderProviderInterface,
     Feature\ConfigProviderInterface,
     Feature\BootstrapListenerInterface
 {
-    public function getAutoloaderConfig()
```
Comment (svycka) [inline src/SlmLocale/Module.php]: I am not sure about this because I never used. Is it safe to remove?

**Comment by basz** [comment]
I don't use it either. I think most devs wont as composer takes care of autoloading. I don't mind dropping it. (Do zf3 modules still support it anyway?)

File: src/SlmLocale/Module.php [hunk 2]
```diff
@@ -44,27 +44,14 @@
...
-                'namespaces' => [
-                    __NAMESPACE__ => __DIR__ . '/src/' . __NAMESPACE__,
-                ],
-            ],
-        ];
-    }
-
     public function getConfig()
     {
         return include __DIR__ . '/config/module.config.php';
```
Comment (svycka) [inline src/SlmLocale/Module.php]: whoops forgot to update this :( and this file had zero tests...

Comment (basz) [reply src/SlmLocale/Module.php]: PITA

**Pull request merged**